.github/workflows/release-teleport-usage.yaml

```diff
@@ -0,0 +1,31 @@
+name: teleport-usage-release
```

Comment:
Can we use a more descriptive name here? Maybe `build-usage-image`?

Reply:
Renamed in https://github.com/gravitational/teleport/pull/20621/commits/d0fe4cc69ea2b4d4e9a1aece72a2697407384555.